frontend/src/app/applications/permit-application-view/temp-outfitter-details.component.html

```diff
@@ -69,8 +69,10 @@ <h2 class="app-item-label padding-right-1">File uploads</h2>
 
       <h2 class="app-item-label padding-right-1" *ngIf="application.tempOutfitterFields.advertisingURL || application.tempOutfitterFields.advertisingDescription">Advertising</h2>
       <div *ngIf="application.tempOutfitterFields.advertisingURL">
-        <h3>Promotional website:</h3>
-        <a href="{{ application.tempOutfitterFields.advertisingURL | UrlPipe }}">{{ application.tempOutfitterFields.advertisingURL }}</a>
+        <h3>Promotional website:<br>
+        <span id="location-hint-text" class="help-text usa-form-hint">WARNING: Applicant URL will open a new window</span>
+      </h3>
+        <a title="WARNING: Applicant URL will open a new window" href="{{ application.tempOutfitterFields.advertisingURL | UrlPipe }}" target="_blank">{{ application.tempOutfitterFields.advertisingURL }}</a>
```

Comment:
```suggestion
        <a href="{{ application.tempOutfitterFields.advertisingURL | UrlPipe }}" target="_blank">{{ application.tempOutfitterFields.advertisingURL }}</a>
```

We don't need the `title` attribute here since the content is available just above it.

Reply:
Makes sense. It has been removed and re-pushed!